src/views/Makeups.test.jsx

```diff
@@ -0,0 +1,466 @@
+    rating: 4.0,
+    category: 'cream',
+    product_type: 'blush',
+    tag_list: ['Gluten Free'],
+    created_at: '2016-10-01T18:34:18.458Z',
+    updated_at: '2017-12-23T21:08:40.580Z',
+    product_api_url: 'https://makeup-api.herokuapp.com/api/v1/products/415.json',
+    api_featured_image:
+      '//s3.amazonaws.com/donovanbailey/products/api_featured_images/000/000/415/original/data?1514063320',
+    product_colors: [],
+  },
+  {
+    id: 414,
+    brand: 'maybelline',
+    name: 'Maybelline Dream Bouncy Blush',
+    price: '11.99',
+    price_sign: null,
+    currency: null,
+    image_link:
+      'https://d3t32hsnjxo7q6.cloudfront.net/i/51eacb9efebbaee39399e65ffe3d9416_ra,w158,h184_pa,w158,h184.png',
+    product_link: 'https://well.ca/products/maybelline-dream-bouncy-blush_50689.html',
+    website_link: 'https://well.ca',
+    description:
+      'Now, blush has bounce! Freshest flush ever:• New bouncy texture is formulated with silicone elastomers• Lightweight like a powder, yet melts seamlessly into skin like a cream giving you a fresh flush• Dermatologist tested• Allergy tested• Non-comedogenic\n                    \n                    \n                    \n                        \n                        For best results: With your fingertips, blend a small amount of\n blush onto the apples of your cheeks, applying from your cheekbones to \nyour temples. To build colour intensity, apply more blush. \n                    ',
+    rating: 4.5,
+    category: 'cream',
+    product_type: 'blush',
+    tag_list: [],
+    created_at: '2016-10-01T18:34:17.251Z',
+    updated_at: '2017-12-23T21:08:40.418Z',
+    product_api_url: 'https://makeup-api.herokuapp.com/api/v1/products/414.json',
+    api_featured_image:
+      '//s3.amazonaws.com/donovanbailey/products/api_featured_images/000/000/414/original/open-uri20171223-4-cjvuw0?1514063320',
+    product_colors: [
+      {
+        hex_value: '#ecc3d5',
+        colour_name: 'Fresh Pink ',
+      },
+      {
+        hex_value: '#ee9cb1',
+        colour_name: 'Hot Tamale ',
+      },
+      {
+        hex_value: '#ecbaba',
+        colour_name: 'Peach Satin ',
+      },
+      {
+        hex_value: '#c25159',
+        colour_name: 'Plum Wine ',
+      },
+    ],
+  },
+];
+
+const server = setupServer(
+  rest.get(
+    'https://makeup-api.herokuapp.com/api/v1/products.json?product_category=cream&product_type=blush',
```

Comment:
We don't need to include the query params in the URL that we are mocking (MSW will ignore them and warn you). We should however validate that our application has made the request with the proper params. 
https://mswjs.io/docs/recipes/request-assertions